packages/circle-react-elements/src/components/ChainLabel/ChainLabel.tsx

```diff
@@ -0,0 +1,19 @@
+import { BlockchainLabelMap } from '../../lib/constants';
+import { SdkType } from '../../lib/types';
+import { ChainIcon } from '../ChainIcon';
+
+export interface ChainSelectProps {
+  /** The blockchain network */
+  blockchain: SdkType.Blockchain;
```

Comment:
Blockchain type you can use directly from circle sdk, like
```
import type { Blockchain } from '@circle-fin/developer-controlled-wallets';
```
in general I'd try to keep components in this package as simple as possible, with as little dependencies as possible

The same with `BlockchainLabelMap`, keep the definition where it is used if not reused.

Reply:
fixed